releases/release-7.5.5.md

```diff
@@ -0,0 +1,233 @@
+    - (dup): release-8.5.0.md > 错误修复> TiDB - 修复 `INDEX_HASH_JOIN` 在异常退出时可能卡住的问题 [#54055](https://github.com/pingcap/tidb/issues/54055) @[wshwsh12](https://github.com/wshwsh12)
+    - 修复 DXF 相关系统表查询可能导致升级异常的问题 [#49263](https://github.com/pingcap/tidb/issues/49263) @[D3Hunter](https://github.com/D3Hunter)
+    - 修复 DDL 内部事务因 "GC life time is shorter than transaction duration" 导致添加索引失败的问题 [#57043](https://github.com/pingcap/tidb/issues/57043) @[tangenta](https://github.com/tangenta)
+    - 修复当 exchange partition 遇到不合法的行时导致 info schema 全量加载的问题 [#56685](https://github.com/pingcap/tidb/issues/56685) @[D3Hunter](https://github.com/D3Hunter)
+    - 修复当 tidb_ddl_enable_fast_reorg 以及 new collation 开启时，未正确处理 collation 导致数据索引不一致的问题 [#58036](https://github.com/pingcap/tidb/issues/58036) @[djshow832](https://github.com/djshow832)
+    - 修复添加索引期间 plan cache 使用了错误的 schema 导致数据索引不一致的问题 [#56733](https://github.com/pingcap/tidb/issues/56733) @[wjhuang2016](https://github.com/wjhuang2016)
+    - 修复在升级期间执行 alter table tiflash replica 导致 TiDB 节点挂掉的问题 [#57863](https://github.com/pingcap/tidb/issues/57863) @[tangenta](https://github.com/tangenta) <!--tw@hfxsd: the following 6 notes-->
+    - 修复 INFORMATION_SCHEMA.columns 查询性能回退的问题 [#58184](https://github.com/pingcap/tidb/issues/58184) @[lance6716](https://github.com/lance6716)
+    - (dup): release-8.1.2.md > 错误修复> TiDB - 修复查询 TiFlash 系统表中默认超时时间过短的问题 [#57816](https://github.com/pingcap/tidb/issues/57816) @[JaySon-Huang](https://github.com/JaySon-Huang)
+    - (dup): release-8.1.2.md > 错误修复> TiDB - 修复 `default_collation_for_utf8mb4` 变量的值没有对 `SET NAMES` 语法生效的问题 [#56439](https://github.com/pingcap/tidb/issues/56439) @[Defined2014](https://github.com/Defined2014)
+    - (dup): release-8.1.2.md > 错误修复> TiDB - 修复手动删除 `mysql`.`timer` 表中的定时器时，TTL 内部协程可能 panic 的问题 [#57112](https://github.com/pingcap/tidb/issues/57112) @[lcwangchao](https://github.com/lcwangchao)
+    - 修复在一些情况下，使用 `ALTER TABLE` 语法将普通表转换为分区表时，因为检查不充分导致数据出现错误的问题 [#55721](https://github.com/pingcap/tidb/issues/55721) @[mjonss](https://github.com/mjonss)
+    - 修复 tidb_gogc_tuner_max_value/tidb_gogc_tuner_min_value 因为空的 max value 导致错误的 warning 信息 [#57889](https://github.com/pingcap/tidb/issues/57889) @[hawkingrei](https://github.com/hawkingrei)
+    - 修复一些情况下，TiDB 内部协程会出现 Data Race 错误的问题 [#57798](https://github.com/pingcap/tidb/issues/57798) [#56053](https://github.com/pingcap/tidb/issues/56053) @[fishiu](https://github.com/fishiu) @[tiancaiamao](https://github.com/tiancaiamao)
+    - 更新 golang-jwt/jwt 来规避潜在的安全风险 [#57135](https://github.com/pingcap/tidb/issues/57135) @[hawkingrei](https://github.com/hawkingrei)
+    - 修复在一些情况下，使用 `ALTER TABLE` 语法将聚簇索引表转换为分区表时，并发写入会造成重复数据的问题 [#57510](https://github.com/pingcap/tidb/issues/57510) @[mjonss](https://github.com/mjonss)
```

Comment:
```suggestion
    -  Fix the issue that under certain circumstances, when using the `ALTER TABLE` statement to convert a normal table to a partitioned table, insufficient checking results in incorrect data [#55721](https://github.com/pingcap/tidb/issues/55721) @[mjonss](https://github.com/mjonss)
    - Fix the issue that when setting `tidb_gogc_tuner_max_value` and `tidb_gogc_tuner_min_value`, the maximum value is null, which leads to an incorrect warning message [#57889](https://github.com/pingcap/tidb/issues/57889) @[hawkingrei](https://github.com/hawkingrei)
    - Fix the issue of data race in TiDB internal concatenation in some cases [#57798](https://github.com/pingcap/tidb/issues/57798) [#56053](https://github.com/pingcap/tidb/issues/56053) @[fishiu](https://github.com/fishiu) @[tiancaiamao](https://github.com/tiancaiamao)
    - Update `golang-jwt` and `jwt` to avoid potential security risks [#57135](https://github.com/pingcap/tidb/issues/57135) @[hawkingrei](https://github.com/hawkingrei)
    - Fix the issue that concurrent writes result in duplicate data when converting a clustered indexed table to a partitioned table using the `ALTER TABLE` statement under certain circumstances [#57510](https://github.com/pingcap/tidb/issues/57510) @[mjonss](https://github.com/mjonss)
```